src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -406,9 +406,10 @@ Whether Harvesting (OAI) service is enabled
          */
         InheritParentRoleAssignments,
         
-        /** Make Data Count Logging and Display */
+        /** Make Data Count Logging, Display, and Start Date */
         MDCLogPath, 
         DisplayMDCMetrics,
+        MDCStartDate,
```

Comment:
Can `:MDCStartDate` be changed from a database setting to MPCONFIG?

And can we please have a release not about the new setting?